plugins/inventory/proxmox.py

```diff
@@ -339,7 +339,8 @@ def _populate(self):
 
                 for member in self._get_members_per_pool(pool['poolid']):
                     if member.get('name'):
-                        self.inventory.add_child(pool_group, member['name'])
+                        if not member['template']:
```

Comment:
Is `'template'` always present, or is it better to write `if not member.get('template'):` here?

Reply:
You're right, it isn't always present :-) fixed that!